tasks.py

```diff
@@ -14,7 +14,7 @@
 import invoke
 
 NOTEBOOK_VERSION = '5.1.0' # the notebook version whose LESS we will use
-NOTEBOOK_CHECKSUM = 'cf7094e06ab65d7cf8c43fefdf647736ef8a5da880ce4c8f351e10f888aaab9a' # sha256 checksum of notebook tarball
+NOTEBOOK_CHECKSUM = 'c35fccb55250688303f6ef35c3870f6f5a8df885ff4d9e5cfe1176a87be7fdda' # sha256 checksum of notebook tarball
```

Comment:
Needed to boot travis into action.

Reply:
@minrk  bumped the SHA here: https://github.com/jupyter/nbviewer/pull/726/files. Not sure why it doesn't match. A new tarball was uploaded to pypi? Something untoward?

Reply:
Confirming I get what you get:

```
 ❯❯❯ pip download notebook==5.1.0 --no-deps -d . --no-binary :all:
Collecting notebook==5.1.0
  Downloading notebook-5.1.0.tar.gz (12.4MB)
    100% |████████████████████████████████| 12.4MB 48kB/s
  Saved ./notebook-5.1.0.tar.gz
Successfully downloaded notebook
~ ❯❯❯ shasum -a 256 notebook-5.1.0.tar.gz
c35fccb55250688303f6ef35c3870f6f5a8df885ff4d9e5cfe1176a87be7fdda  notebook-5.1.0.tar.gz
```

Reply:
I got the checksum from [PyPI](https://pypi.org/project/notebook/5.1.0/#files), and managed to copy the checksum of the wheel instead of the sdist. Not sure how I didn't catch that.